database/config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
It would be a good idea to put theses in a git ignore so your passwords are not available.